### PR TITLE
Documentation cleanup 5

### PR DESCRIPTION
**Pull request by oscargus**
#### Brief description of what is fixed or changed

Primarily LaTeX:ifying stuff, but some general fixes as well.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/stats/joint_rv_types.py [hunk 1]
```diff
@@ -713,9 +713,9 @@ def GeneralizedMultivariateLogGamma(syms, delta, v, lamda, mu):
     ====
 
     If the GeneralizedMultivariateLogGamma is too long to type use,
-    `from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma as GMVLG`
+    ``from sympy.stats.joint_rv_types import GeneralizedMultivariateLogGamma as GMVLG``
```
Comment (ThePauliPrinciple) [inline sympy/stats/joint_rv_types.py:716]: Is this instruction really needed? Users can import it as any name as far as I am concerned.

**Comment by ThePauliPrinciple** [comment]
Changes make sense, looks good to me. Seems like the `.mailmap` needs to be updated.

File: sympy/stats/joint_rv_types.py (hunk 1 above)
Comment (oscargus) [reply sympy/stats/joint_rv_types.py:716]: I wasn't convinced either. But at least it should be properly formatted...

**Comment by oscargus** [comment]
> Seems like the `.mailmap` needs to be updated.

Have to figure out why that email shows up here... Should probably fix one of my computers to use my private email...

**Comment by oscarbenjamin** [comment]
> Have to figure out why that email shows up here...

There's only one commit here and that's the email address it shows:
https://github.com/sympy/sympy/commit/c9f6ba02dfa7ccc1b8b408d5b07d1ab96f11eb0d.patch
I'd say whichever computer you used to make this PR.

**Comment by oscargus** [comment]
I have force pushed from different computers and apparently I had things not correctly setup on the first and the address is not changed "automatically" when force pushing from the one with correct email. Now I have changed it manually though (I hope).

**Pull request merged**